Automattic/pocket-casts-android
# Display Completion Rate story

## Description

This adds the Completion Rate story UI.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_4018

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the Completion Rate story and verify it.

## Screenshots or Screencast 

### Preview

![Screenshot 2024-10-18 at 09 25 30](https://github.com/user-attachments/assets/f64e2c49-63a1-44a8-88c7-a6f7bb9cd9f0)

### Animation

https://github.com/user-attachments/assets/514a867f-b459-4674-a799-00faecb7bd47

## Checklist
 
#### I have tested any UI changes...
